README.md

```diff
@@ -20,26 +20,24 @@ i.e.
 - `PORT` - Port Number
 - `DATABASE_URL` - URL to the running Postgres instance/db
 
-npm init - to create a package.json file
-npm install nodemon - lets you run your server locally (may need to be installed multiple times while working on a project)
-npm install prettierrc - a linter that only seems to exist to break stuff
+npx create-react-app resty --template typescript 
+npm install
 npm install jest - lets you use jest to test your code
-npm install supertest - lets you use some special part of jest to test your code (something to do with grouping things together and/or "describe" look it up!
-npm install express - lets you instantiate an express server
+
 
 
 #### How to initialize/run your application (where applicable)
 
-- e.g. `npm start`
+npm start
 
 #### How to use your library (where applicable)
 
 #### Tests
-
+<!--
 - How do you run tests?
 - Any tests of note?
 - Describe any tests that you did not complete, skipped, etc
 
 #### UML
 
-Link to an image of the UML for your application and response to events
\ No newline at end of file
+Link to an image of the UML for your application and response to events -->
```

Comment:
You can remove the .env part. If you want and if it would help, you can add a section with details about each lab and what was required, but only if this would help you.